orchestra/testlists/testlists.go

```diff
@@ -77,10 +77,10 @@ func (c *Client) Do(
 	if countryCode != "" {
 		query.Set("probe_cc", countryCode)
 	}
-	if (limit > 0) {
+	if limit > 0 {
 		query.Set("limit", fmt.Sprintf("%d", limit))
 	}
-	if (len(c.EnabledCategories) > 0) {
+	if len(c.EnabledCategories) > 0 {
```

Comment:
Is this done automatically by some linting tool? Maybe we should add it to the CI to check code style stuff.

Reply:
`gofmt -s -w`

We have a check that does that automatically for us as a badge in the readme. It checks for several Go linting and quality aspects: https://goreportcard.com/report/github.com/ooni/probe-engine